### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by sh0hei**

### CODE REVIEW

**Review by yourzbuddha — APPROVED** [review]
Looks good!

**Comment by yourzbuddha** [comment]
@sh0hei is it gonna be merged?

File: project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version=0.13.16
+sbt.version=1.5.5
```
Comment (yourzbuddha) [inline project/build.properties:1]: You can use 1.7.1

**Review by yourzbuddha — CHANGES_REQUESTED** [review]

**Comment by noelwelsh** [comment]
Thanks!

**Pull request merged**